scripts/items/sacrifice_torque.lua

```diff
@@ -0,0 +1,44 @@
+-----------------------------------
+-- ID: 15528
+-- Sacrifice Torque
+-- Pet mod via latent effect
+-- Latents are added to player before OnItemEquip is called
+-- Latents are removed from player before OnItemUnequip is called
+-----------------------------------
+local itemObject = {}
+local listenerPrefix = 'PET_MOD_LATENT'
+local latentPetId = nil
+local latentMods =
+{
+    { xi.mod.ATT, 3 },
+}
+
+itemObject.onItemCheck = function(target)
+    return 0
+end
+
+itemObject.onItemEquip  = function(user, item)
+    local listenerName = fmt('{}_{}', listenerPrefix, item:getID())
+    if user:hasAllLatentsActive(xi.slot.NECK) then
+        user:setLocalVar(listenerName, 1)
```

Comment:
when does this get removed?

Reply:
(or set to 0)

Reply:
Oops it should get reset on unequip regardless if you have a pet, for consistency

Reply:
Fixed.

FWIW i'm not sure there's a scenario where that would have caused a problem, since this new logic for the handful of "specific-avatar latent mods" applies mods to pets. If this latent were inactive (not setting the localvar), then you wouldn't have a pet for the localvar to matter.

Good catch regardless. Better to be explicit than make assumptions